_rules/focusable-no-keyboard-trap-80af7b.md

```diff
@@ -54,6 +54,8 @@ There are no accessibility support issues known.
 
 ## Background
 
+This rule only requires navigation in one direction (either forward or backward), not both, and not a specific one. It is clear that not being able to escape a focus trap in any direction is a failure of [Success Criterion 2.1.2 No keyboard trap][sc212]. However, it is less clear that being able to escape in only one direction is enough to satisfy it. If [Success Criterion 2.1.2 No keyboard trap][sc212] requires the possibility to escape the trap in a specific way (e.g. forward [standard keyboard navigation](#standard-keyboard-navigation)) or in both directions, this rule may pass while the criterion is not satisfied.
```

Comment:
[sc212] is not defined at the end of the file

Reply:
thanks @carlosapaduarte, I added it in.